File: check_plugins.mjs
// bun has built-in sqlite
import { Database } from "bun:sqlite";
import path from "path";
const dbPath = path.join(process.env.LOCALAPPDATA, "Peekoo", "peekoo", "peekoo.sqlite");
const db = new Database(dbPath, { readonly: true });
const rows = db.query("SELECT plugin_key, enabled FROM plugins ORDER BY plugin_key").all();
rows.forEach(r => console.log(r.plugin_key, ":", r.enabled));
db.close();

File: check_plugins.py
import sqlite3, os, sys
db = os.path.join(os.environ['LOCALAPPDATA'], 'Peekoo', 'peekoo', 'peekoo.sqlite')
print(f"DB: {db}")
conn = sqlite3.connect(db)
cur = conn.cursor()
cur.execute("SELECT plugin_key, enabled FROM plugins ORDER BY plugin_key")
rows = cur.fetchall()
for row in rows:
    print(f"  {row[0]}: enabled={row[1]}")
conn.close()
